source/beamline/PGM/PGMBeamline.cpp

```diff
@@ -77,7 +91,10 @@ void PGMBeamline::setupControlsAsDetectors()
 
 void PGMBeamline::setupExposedControls()
 {
-
+    // Storage ring current
+    addExposedControl(ringCurrent_);
```

Comment:
These are not typically exposed controls.  Exposed controls are things that you plan to expose to scan controllers to change.